### PR TITLE
Normative: Emit reference information when calendarName = 'always'

### PR DESCRIPTION
**Pull request by jessealama**
closes #1971

### CODE REVIEW

**Comment by ptomato** [comment]
This needs a corresponding change in the spec text.

**Comment by jessealama** [comment]
updated spec text

File: spec/plainmonthday.html [hunk 1]
```diff
@@ -405,7 +405,7 @@ <h1>TemporalMonthDayToString ( _monthDay_, _showCalendar_ )</h1>
         1. Let _day_ be _monthDay_.[[ISODay]] formatted as a two-digit decimal number, padded to the left with a zero if necessary.
         1. Let _result_ be the string-concatenation of _month_, the code unit 0x002D (HYPHEN-MINUS), and _day_.
         1. Let _calendarID_ be ? ToString(_monthDay_.[[Calendar]]).
-        1. If _calendarID_ is not *"iso8601"*, then
+        1. If _showCalendar_ is not *"always"* or if _calendarID_ is not *"iso8601"*, then
```
Comment (ptomato) [inline spec/plainmonthday.html:408]:
```suggestion
        1. If _showCalendar_ is *"always"* or if _calendarID_ is not *"iso8601"*, then
```
?

**Review by ptomato — CHANGES_REQUESTED** [review]

File: spec/plainyearmonth.html [hunk 2]
```diff
@@ -684,7 +684,7 @@ <h1>TemporalYearMonthToString ( _yearMonth_, _showCalendar_ )</h1>
         1. Let _month_ be _yearMonth_.[[ISOMonth]] formatted as a two-digit decimal number, padded to the left with a zero if necessary.
         1. Let _result_ be the string-concatenation of _year_, the code unit 0x002D (HYPHEN-MINUS), and _month_.
         1. Let _calendarID_ be ? ToString(_yearMonth_.[[Calendar]]).
-        1. If _calendarID_ is not *"iso8601"*, then
+        1. If _showCalendar_ is not *"always" or if *_calendarID_ is not *"iso8601"*, then
```
[the remark this answers is not in the document]
Comment (jessealama) [reply spec/plainyearmonth.html:687]: quite right, thanks!

[the remark this answers is not in the document]
Comment (jessealama) [reply spec/plainyearmonth.html]: oops, yes -- done

**Review by ptomato — APPROVED** [review]
:+1:

Will keep this marked as a draft until presented at TC39.

**Comment by ptomato** [comment]
This achieved consensus at the March 2022 TC39 plenary meeting.

**Pull request merged**